tests/base_tests.py

```diff
@@ -372,10 +392,16 @@ def test_X_no_rows_error(
         ):
             x.fit(df, df["a"])
 
+    @pytest.mark.parametrize(
+        ("minimal_dataframe_lookup", "get_series_init"),
```

Comment:
since this is just for this test it might make more sense to parameterise the series init methods here rather than parameterize the call to the fixture?

Reply:
yeah fair enough, implemented as an if/else inside the test https://github.com/lvgig/tubular/pull/328/commits/aef866eecd25b4db84d12765cfb8684d09996ae1#diff-09e907e492bc399633765530bd5f9a6d3a4d867e068002e04b79d26bd6f17f68